hq/test/logic/SnykDisplayTest.scala

```diff
@@ -0,0 +1,211 @@
+package logic
+
+import model._
+import org.scalatest.{FreeSpec, Matchers}
+import utils.attempt.AttemptValues
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class SnykDisplayTest extends FreeSpec with Matchers with AttemptValues {
+
+  "find organisationId" in {
+    val organisationId = SnykDisplay.getOrganisationId(SnykDisplayTest.mockGoodOrganisationResponse, Organisation("guardian"))
+    organisationId.value() shouldBe "1111111111"
+  }
+
+  "fail to find organisationId (nice)" in {
+    val organisationId = SnykDisplay.getOrganisationId(SnykDisplayTest.mockBadResponseWithMessage, Organisation("guardian"))
+    organisationId.isFailedAttempt shouldBe true
+    organisationId.getFailedAttempt.failures.head.friendlyMessage shouldBe "Could not read Snyk response (some nice error)"
+  }
+
+  "fail to find organisationId (not nice)" in {
+    val organisationId = SnykDisplay.getOrganisationId(SnykDisplayTest.mockBadResponseWithoutMessage, Organisation("guardian"))
+    organisationId.isFailedAttempt shouldBe true
+    organisationId.getFailedAttempt.failures.head.friendlyMessage shouldBe """Could not read Snyk response ({"toughluck": "no use"})"""
+  }
+
+  "find project id list" in {
+    val projects = SnykDisplay.getProjectIdList(SnykDisplayTest.mockGoodProjectResponse)
+    !projects.value().exists(p => p.name == "project1") shouldBe false
```

Comment:
ScalaTest has nice ways to test properties of collections. As well as being terser and easier to read, you'll get better error messages when it fails. In this case it's not so simple because you're looking for a nested property, but I'll put this link here for your perusal!

http://www.scalatest.org/user_guide/using_matchers